bochs/Makefile.in

```diff
@@ -349,6 +350,11 @@ cpu/fpu/libfpu.a::
 	$(MAKE) $(MDEFINES) libfpu.a
 	@CD_UP_TWO@
 
+cpu/softftloat3e/libsoftftloat.a::
```

Comment:
should be `cpu/softfloat3e/libsoftfloat.a::`
(`ftft` breaks build)

Reply:
Thanks !